Signal/src/ViewControllers/SignalAttachment.swift

```diff
@@ -300,9 +261,13 @@ class SignalAttachment: NSObject {
                 return audioAttachment(data : data, dataUTI : dataUTI)
             }
         }
-        // TODO: We could handle generic attachments at this point.
 
-        return nil
+        let dataUTI = pasteboardUTISet[pasteboardUTISet.startIndex]
+        guard let data = dataForFirstPasteboardItem(dataUTI:dataUTI) else {
+            Logger.verbose("\(TAG) Missing expected pasteboard data for UTI: \(dataUTI)")
```

Comment:
nit: This *seems* like an error condition? 

If so, I'd say a Logger.error and assertionFailure are in order.